legend-engine-xt-relationalStore-spanner-jdbc-shaded/pom.xml

```diff
@@ -0,0 +1,338 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>legend-engine</artifactId>
+        <groupId>org.finos.legend.engine</groupId>
+        <version>3.22.4-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>legend-engine-xt-relationalStore-spanner-jdbc-shaded</artifactId>
+    <name>Legend Engine - XT - Relational Store - Spanner - Execution</name>
+
+    <properties>
+        <maven-shade-plugin.shadedPattern>org.finos.legend.engine.spanner.jdbc.shaded</maven-shade-plugin.shadedPattern>
+    </properties>
+    
+    <dependencies>
+        <dependency>
+            <groupId>com.google.cloud</groupId>
+            <artifactId>google-cloud-spanner-jdbc</artifactId>
+            <version>2.7.12</version>
+            <classifier>single-jar-with-dependencies</classifier>
+            <!--excluding everything cause everything would be in full jar and shaded-->
```

Comment:
Is there a way to exclude everything with a single directive, instead of explicit listing? When we increment the version, and miss excluding a newly added dependency, then it might be problem

Reply:
It seems that templates are also working:

            <exclusions>
                <exclusion>
                    <groupId>*</groupId>
                    <artifactId>*</artifactId>
                </exclusion>
            </exclusions>